PowerEditor/src/MISC/PluginsManager/Notepad_plus_msgs.h

```diff
@@ -549,7 +549,8 @@ enum Platform { PF_UNKNOWN, PF_X86, PF_X64, PF_IA64, PF_ARM64 };
 	// void* NPPM_GETBOOKMARKID(0, 0)
 	// Returns the bookmark ID
 
-
+	#define NPPM_CLOSEDOC (NPPMSG + 112)

```

Comment:
The message is wrong named (should be `NPPM_INTERNAL_CLOSEDOC` ) and wrong placed (should be in "resource.h").
Here is opened API for plugins - once a new API is added here, it should be maintained & available. Since close doc API has never been asked, let's not add it.

Reply:
@donho,

I'm wondering if you also meant that the new `closeDoc()` function wasn't necessary and an existing Close function could be used instead.

Reply:
@Yaron10 
> I'm wondering if you also meant that the new closeDoc() function wasn't necessary and an existing Close function could be used instead.

I mean only for the API `NPPM_CLOSEDOC` which should be `NPPM_INTERNAL_CLOSEDOC`. But if you see any existent function which can replace closeDoc() function, please let me know.

Reply:
@donho,

I did not have anything concrete in mind. Just a vague thought.

Thank you and @robotboy655 for your work. 👍